cmd/apps/nfs_app.go

```diff
@@ -9,16 +9,16 @@ import (
 	"github.com/alexellis/arkade/pkg"
 	"github.com/alexellis/arkade/pkg/apps"
 	"github.com/alexellis/arkade/pkg/config"
-	"github.com/alexellis/arkade/pkg/k8s"
 	"github.com/alexellis/arkade/pkg/types"
 	"github.com/spf13/cobra"
 )
 
 func MakeInstallNfsProvisioner() *cobra.Command {
 	var nfsProvisionerApp = &cobra.Command{
-		Use:          "nfs-client-provisioner",
-		Short:        "Install nfs client provisioner",
-		Long:         "Install nfs client provisioner to create dynamic persistent volumes",
+		Use:          "nfs-subdir-external-provisioner",
```

Comment:
What a long time. Glad you have the alias.

Perhaps one more alias of `nfs-provisioner` could work?

Reply:
Sure, good idea, I've added it.